Ehr_OOP.py

```diff
@@ -0,0 +1,211 @@
+"""Bio stats assignment #OOP Mohammad Anas."""
+
+
+from datetime import datetime
+from typing import Union
+
+
+class Patient:
+    """Store a patient data."""
+
+    def __init__(
+        self,
+        PatientID: str,
+        gender: str,
+        DOB: Union[str, datetime],
+        race: str,
+        labs: list[str],
+    ) -> None:
+        """Instantiate Patient Class."""
+        self.patientID: str = PatientID  # O(1)
+        self.gender: str = gender  # O(1)
+        self.DOB: datetime = datetime.strptime(DOB, "%Y-%m-%d %H:%M:%S.%f")  # O(2)
```

Comment:
```suggestion
        self.dob: datetime = datetime.strptime(dob, "%Y-%m-%d %H:%M:%S.%f")  # O(2)
```
PEP8 style guidance would have variable and attribute names start with a lowercase letter. We typically capitalize acronyms as if they were ordinary words.